File: Problems/Problem_8/problem_8.py
str = input("Enter string: ")
i=0
result = False
while(i<int(len(str)/2)):
    if str[i].lower() == str[len(str)-i-1].lower():
        result = True
    else:
        result = False
        break
    i = i+1

if result:
    print(f"{str} is palindrome")
else:
    print(f"{str} is not palindrome")

File: Problems/Problem_8/problem_8.c
#include<stdio.h>
#include<string.h>
int main()
{
    char set[50],n,i,a=0;
    printf("Enter string:");
    scanf("%[^\n]%*c",set);
    n=strlen(set);
    for(i=0;i<(n/2);i++)
    {
        if(set[i]==set[n-i-1])
        {
            a++;
        }
    }
    if(a==i)
    {
        printf("%s is a palindrome");
    }
    else
    {
        printf("%s is not a palindrome");
    }

}